okapi-common/src/test/java/org/folio/okapi/common/ModuleIdTest.java

```diff
@@ -84,5 +85,10 @@ public void test2() {
     assertTrue(module_1_10000.hasSemVer());
     assertFalse(module_1_10000.hasPreRelease());
     assertTrue(module_1_10000.hasNpmSnapshot());
+
+    ModuleId module = new ModuleId("module");
+    assertFalse(module.hasSemVer());
+    assertFalse(module.hasPreRelease());
+    assertFalse(module.hasNpmSnapshot());
```

Comment:
This new test should be in a separate unit test method, for example `withoutSemVer()`.

Reply:
Thanks, this has been resolved.